cloudferry_devlab/cloudferry_devlab/tests/README.md

```diff
@@ -0,0 +1,44 @@
+Directory with tests for Cloudferry's devlab
+==========
+
+## Prerequisite action
+
+Prepare small tenant:
+- vms
+- networks
+- subnets
+- users
+- keypairs
+- tenants
+- modifies tenants quotas
+- upload images and snapshot
+- creates flavors
+- creates volumes
```

Comment:
I would suggest to instead give overview of how to run `generate_load.py` is used and give a high-level overview of what types of objects it generates